primitives/core/src/bounded/bounded_vec.rs

```diff
@@ -53,6 +53,24 @@ pub trait TruncateFrom<T> {
 	fn truncate_from(unbound: T) -> Self;
 }
 
+pub trait Min<T> {
```

Comment:
Huh, Rust has no Min/Max traits?   
I dont think we should add some, maybe `Ord + Eq` is enough.  
Since if we add these traits, we would have to implement them for every primitive type.